nixos/tests/postfixadmin.nix

```diff
@@ -1,6 +1,6 @@
 import ./make-test-python.nix ({ pkgs, ...} : {
   name = "postfixadmin";
-  meta = with pkgs.stdenv.lib.maintainers; {
```

Comment:
@siraben Not sure how we missed that.

Reply:
Same, It must have been added after the treewide PRs, but I haven't checked.

Reply:
Perhaps we could add some sort of ofBorg check or similar for stdenv.lib.

Reply:
@SuperSandro2000 #138918